src/DynamoCoreWpf/NodeViewCustomization/NodeViewCustomizations.cs

```diff
@@ -12,7 +13,7 @@ public NodeViewCustomizations(IDictionary<Type, IEnumerable<Type>> customization
             customizations = customizationMap ?? new Dictionary<Type, IEnumerable<Type>>();
         }
 
-        public IDictionary<Type, IEnumerable<Type>> GetCustomizations()
+        public IDictionary<Type, IEnumerable<Type>> GetCustomizations(ILogger logger)
```

Comment:
I guess this feels a little funny - that we don't do anything with the logger here...